services/tests/test_report.py

```diff
@@ -3727,10 +3727,6 @@ def test_create_report_upload(self, dbsession):
         assert res.totals is None
         assert res.upload_extras == {}
         assert res.upload_type == "uploaded"
-        assert len(res.flags) == 1
```

Comment:
This no longer gets evaluated by this test/method since we got rid of the override